### ISSUE TITLE
error message when fixing allometric exponent

### ISSUE THREAD

**Issue by sjtpharm** [nlmixrdevelopment/nlmixr#83]
Error message when trying to fix an exponent in an allometric model (from the nlmixr training at ACoP)
```R
m1 <- function() {
  ini({
    tka <- .5
    tcl <- -3.2
    tv <- -1
    eta.ka ~ 1
    eta.cl ~ 2
    eta.v ~ 1
    beta.wt <- fix(0.75) # error here
    add.err <- 0.1

  })
  model({
    ka <- exp(tka + eta.ka)
    cl <- exp(tcl + eta.cl + beta.wt*logWT70)
    v <- exp(tv + eta.v)
    linCmt() ~ add(add.err)
  })
}
```
Error in configsaem(model = model, data = dat, inits = uif$saem.init,  : 
  FIXED pars cannot have ETA

**Comment by mattfidler** [comment]
Note, you can use FOCEi
```R
d  <- theo_sd;
d$logWT70  <- log(d$WT/70)
fit  <- nlmixr(m1,d,"focei")
```

```R
## Using the development branch
>fit
-- nlmixr FOCEi fit ----------------------------------------------------------------------------------------------------------------------------------------
          OBJF      AIC      BIC Log-likelihood Condition Number
FOCEi 123.0421 137.0421 157.2217      -61.52105         94.45787

-- Time (sec; fit$time): -----------------------------------------------------------------------------------------------------------------------------------
        setup optimize covariance table
elapsed 0.383    4.259      4.259  0.03

-- Population Parameters (fit$parFixed): -------------------------------------------------------------------------------------------------------------------
         Parameter   Est.     SE  %RSE Back-transformed(95%CI) BSV(CV%) Shrink(SD)%
tka                 0.473  0.207  43.8        1.6 (1.07, 2.41)    130.%      31.6% 
tcl                  -3.2 0.0901  2.81 0.0406 (0.0341, 0.0485)    29.2%     -2.50% 
tv                 -0.786 0.0395  5.03    0.456 (0.422, 0.493)    16.8%      23.1% 
beta.wt error here   0.75  FIXED FIXED                    2.12                     
add.err             0.704                                0.704                      

  Covariance Type (fit$covMethod): r,s
  No correlations in between subject variability (BSV) matrix
  Full BSV covariance (fit$omega) or correlation (fit$omegaR; diagonals=SDs)
  Distribution stats (mean/skewness/kurtosis/p-value) available in fit$shrink
```

**Comment by mattfidler** [comment]
I think this has been fixed by @wwang-at-github
